src/sparseml/onnx/utils/helpers.py

```diff
@@ -90,7 +90,7 @@ def validate_onnx_file(path: str):
         if not onnx_model.opset_import:
             raise ValueError("could not parse opset_import")
     except Exception as err:
-        raise ValueError("file at {} is not a valid onnx model: {}".format(path, err))
```

Comment:
@dbogunowicz `path` here can be an onnx.ModelProto, which means it was printing out the entire serialized resnet50 model including parameters. same thing for the value error below